docs/4.x/request-parameters.md

```diff
@@ -0,0 +1,73 @@
+---
+category: Develop
+title: Handling Request Parameters
+---
+# Handling Request Parameters
+
+If you need to access a query or post parameter value, use the [`Common::getRequestVar()`](/api-reference/Piwik/Common#getrequestvar) method.
```

Comment:
Can we maybe show the full signature here with parameters? as in `getRequestVar($varName, $varDefault = null, $varType = null, $requestArrayToUse = null)`

Reply:
```suggestion
If you need to access a query or post parameter value, use the [`Common::getRequestVar($varName, $varDefault = null, $varType = null, $requestArrayToUse = null)`](/api-reference/Piwik/Common#getrequestvar) method.
```